.gitignore

```diff
@@ -0,0 +1 @@
+main
```

Comment:
I believe this is the binary produced by the Makefile. If so, it should have a proper name.

Reply:
changed.